serializers/uri-builder.js

```diff
@@ -4,7 +4,17 @@ const fs = require('fs');
 const yaml = require('js-yaml');
 const url = require('url');
 
-const { protocol, hostname } = config.get('server');
+// Configuration file won't exist until deployment.
+// Use dummy values if we can't load the configuration file.
+function getConfigValues() {
```

Comment:
Good idea, but can you use arrow function here as well for the consistency? Also, I think in ECMAScript 6 or above, we should use arrow function as much as we can. https://stackoverflow.com/questions/22939130/when-should-i-use-arrow-functions-in-ecmascript-6